.github/dependabot.yml

```diff
@@ -16,3 +16,11 @@ updates:
     directory: "ruby/autobuilder"
     schedule:
       interval: "daily"
+
```

Comment:
Rogue space.

Reply:
If you're referencing the new line, that was intentional to separate this entry for the Actions ecosystem from the previous four entries for the Cargo ecosystem.  Happy to remove it if it's unhelpful though.

Reply:
Ah that makes sense!